core/src/test/java/io/questdb/test/griffin/SqlOptimiserTest.java

```diff
@@ -1901,6 +1907,20 @@ public void testOrderingOfSortsInSingleTimestampCase() throws Exception {
         });
     }
 
+    @Test
+    public void testPushDownLimitFromChooseToNone() throws Exception {
+        assertMemoryLeak(() -> {
+            ddl(tripsDdl);
+            drainWalQueue();
+            assertPlanNoLeakCheck("select cab_type, vendor_id, pickup_datetime from trips \n" +
+                    "order by pickup_datetime desc, cab_type desc limit 100;", "Sort light lo: 100 partiallySorted: true\n" +
```

Comment:
Would be great to add tests with non-null hi limit here and below just to make sure we didn't break anything there.